controllers/articles.controllers.js

```diff
@@ -15,4 +15,21 @@ const getArticles = (request, response,next) => {
         next(err);
     });
 };
-module.exports = {getArticles};
\ No newline at end of file
+
+const getCommentsByArticleId = (request, response, next) => {
+
+    const { article_id } = request.params;
+    selectCommentsByArticleId(article_id, next)
+    .then((comments) => {
+        if (comments === undefined) return response.status(404).send({status: 404, msg : 'Not found'});
```

Comment:
We should be handling any kind of checking logic (if necessary) in the model before we send this back to the controller :)